docker/Dockerfile

```diff
@@ -17,7 +21,9 @@ USER oommfuser
 WORKDIR /usr/local
 RUN git clone https://github.com/fangohr/oommf.git
 WORKDIR /usr/local/oommf
-RUN make build-with-all
+
+# This Makefile target builds oommf
+RUN make build
```

Comment:
In principle, only fetching the last commit during `clone` would be enough.

Reply:
You mean with a --depth option or similar? Can you make a suggestion?

Reply:
We could use `git clone --depth 1 https://github.com/fangohr/oommf`.

Reply:
done